etools-searchable-multiselection-menu.html

```diff
@@ -217,7 +225,7 @@
       <template is="dom-if" if="{{!multi}}">
         <paper-input
           id="singleSelectionDropdown"
-          class="value-container"
+          class$="value-container [[capitalizeInputValueClass(capitalizeInputShown)]]"
```

Comment:
please change `capitalizeInputValueClass` to `_capitalizeInputValueClass` as it's a method used only inside the element.